specification/cognitiveservices/data-plane/WebSearch/v1.0/WebSearch.json

```diff
@@ -0,0 +1,1235 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Web Search API",
+    "description": "The Web Search API lets you send a search query to Bing and get back search results that include links to webpages, images, and more.",
+    "version": "1.0"
+  },
+  "parameters": {
+    "x-bingapis-sdk": {
+      "name": "X-BingApis-SDK",
+      "description": "Activate swagger compliance",
+      "x-ms-parameter-location": "method",
+      "required": true,
+      "type": "string",
+      "in": "header",
+      "x-ms-enum": {
+        "name": "XBingApisSDK",
+        "modelAsString": true
+      },
+      "enum": [
+        "true"
+      ]
+    }
+  },
+  "host": "api.cognitive.microsoft.com",
+  "schemes": [
+    "https"
+  ],
+  "basePath": "/bing/v7.0",
+  "produces": [
+    "application/json"
+  ],
+  "securityDefinitions": {
+    "apiKeyHeader": {
+      "name": "Ocp-Apim-Subscription-Key",
+      "type": "apiKey",
+      "in": "header"
+    },
+    "apiKeyQuery": {
+      "name": "subscription-key",
+      "type": "apiKey",
+      "in": "query"
+    }
+  },
+  "security": [
+    {
+      "apiKeyHeader": []
+    },
+    {
+      "apiKeyQuery": []
+    }
+  ],
+  "paths": {
+    "/search": {
+      "get": {
+        "summary": "The Web Search API lets you send a search query to Bing and get back search results that include links to webpages, images, and more.",
+        "operationId": "Web_Search",
+        "tags": [
+          "WebSearch"
+        ],
+        "parameters": [
+          {
+            "$ref": "#/parameters/x-bingapis-sdk"
+          },
+          {
+            "name": "Accept",
```

Comment:
In general, specifying an Accept header that doesn't provide any information in addition to what the produces property for the API already provides. The main reason to include it would be to control the generated SDK API surface area, but if you intend to use AutoRest to generate the SDK, I'm not sure under what circumstances an Accept header other then application/json would be applicable...